moveos/moveos-store/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::event_store::EventStore;
+use anyhow::Result;
+use std::sync::Arc;
+
+pub mod event_store;
+pub mod state_store;
+
+#[derive(Clone)]
+pub struct RoochDB {
```

Comment:
Here in the MoveOS crate, it should be the MoveOSDB.

Reply:
> Here in the MoveOS crate, it should be the MoveOSDB.

fixed